fjord/translations/models.py

```diff
@@ -510,9 +434,13 @@ def translate(self, instance, src_lang, src_field, dst_lang, dst_field):
                 metadata=metadata)
             return
 
-        # If the src -> dst isn't a supported pair, log an issue for
-        # metrics purposes and move on.
-        if (lc_src, dst_lang) not in gengo_api.get_language_pairs():
+        # If this is 'standard' tier and if the src -> dst isn't a
+        # supported pair, log an issue for metrics purposes and move
+        # on. We only do this for 'standard' tier since the 'machine'
+        # tier has a different set of pairs which aren't available to
+        # us.
+        if ((self.gengo_tier == 'standard'
```

Comment:
This (and some other things like it) seem like they should be part of the subclasses, instead of mixing all the logic in the super class. Any particular reason the subclasses are so light and the parent class so heavy?